src/main.ts

```diff
@@ -2,6 +2,10 @@ import * as dotenv from 'dotenv';
 import * as fs from 'fs';
 dotenv.config();
 
+import Config from './config';
+import * as Sentry from '@sentry/node';
+Sentry.init({ dsn: Config.sentry.DSN });
```

Comment:
Just wondering how are we going to handle the data from different environments, are we going to have a different `DSN` value for each environment?

Reply:
Nope, same DSN for each environment.  It's pretty easy to tag each environment with an environment variable upon initialization of Sentry which can then be used as a filter in the Sentry UI.  I'm happy to hook that up if you can guide me toward how the code knows which environment it's in.

It's also worth mentioning that a missing DSN is the correct way to tell Sentry not to log anything, so it's perfectly fine for it to remain blank on our local environments.